test/test_api/test_base_api.py

```diff
@@ -143,3 +144,19 @@ def test_pipeline_get_budget(fit_dictionary_tabular, min_budget, max_budget, bud
         assert list(smac_mock.call_args)[1]['ta_kwargs']['pipeline_config'] == default_pipeline_config
         assert list(smac_mock.call_args)[1]['max_budget'] == max_budget
         assert list(smac_mock.call_args)[1]['initial_budget'] == min_budget
+
+
+def test_no_resampling_error(backend):
+    """
+    Checks if an error is raised when trying to construct ensemble
+    using `NoResamplingStrategy`.
+    """
+    BaseTask.__abstractmethods__ = set()
```

Comment:
Btw, I saw it in a last few PRs, but what is it?

Reply:
So, according to @eddiebergman's suggestion, I made `BaseTask` a meta class and since doing that we can't instantiate an object of this class. So, this is a workaround for the tests.